lib/spack/spack/test/compilers/basics.py

```diff
@@ -841,6 +849,8 @@ def test_compiler_flags_use_real_version(working_env, monkeypatch, tmpdir):
     assert flag == '-std=c++0x'
 
 
+@pytest.mark.skipif(sys.platform == 'win32',
+                    reason="Not supported on Windows (yet)")
```

Comment:
Would it make sense to eventually support this on Windows? My guess is that the answer is no and the `reason` string could be changed).